.readme-partials.yaml

```diff
@@ -0,0 +1,101 @@
+introduction: |-
+  > This module provides Cloud Debugger support for Node.js applications.
+  Cloud Debugger is a feature of Google Cloud Platform that lets you debug your
+  applications in production without stopping or pausing your application.
+
+body: |-
+  ## Debugger Agent Settings
+
+  You can customize the behaviour of the automatic debugger agent. See
+  [the agent configuration][config-ts] for a list of possible configuration
+  options. These options can be passed in the `options` object passed to the
+  `start` function.
+
+  ```js
+  require('@google-cloud/debug-agent').start({
+    // .. auth settings ..
+
+    // debug agent settings:
+    allowExpressions: true,
+    serviceContext: {
+      service: 'my-service',
+      version: 'version-1'
+    },
+    capture: { maxFrames: 20, maxProperties: 100 }
+  });
+  ```
+
+  ## Using the Debugger
+
+  Once your application is running (deployed, or elsewhere), you should be able
+  to use the [Debug UI][debug-tab] in your Cloud
+  [developer console][dev-console]. You can find the Debug UI in the
+  'Operations -> Debug' section in the navigation panel, or by simply searching
+  for 'Debug' in the cloud console.
+
+  If you have the source available, you can set a snapshot by clicking in the
```

Comment:
This would flow better as a numbered list of instructions I think. For example:

To take a snapshot with the debugger:

1. Click in the gutter (line number area) or enter a filename and line number in the snapshot panel
2. The debugger inserts a momentary breakpoint at the specified location in your code in the running instance of your application.

Reply:
Done, and yes, it does read easier now.